### PR TITLE
Record organisers’ overrides

### PR DESCRIPTION
**Pull request by gnclmorais**
As an organiser, I’ve ran into situations where I believe an invite or two has been edited by another organiser, but I don’t have a way to know. I’ve also had a student or two telling me they’ve received the attendance email, but their names are not on the list.

Hopefully, this PR will help with it. Now, whenever an organiser adds/removes a person from a workshop, we record their ID so we can connect that editing with their action. This is not the most exact method, but it saves us from recording every single change or creating a table just for this.

| organiser page | new _changes_ page |
|:-:|:-:|
|image|image|

Bonus: this PR also addresses a few other things I found:
- the fancy error screen we have was not allowing the [`better_errors` gem](https://github.com/BetterErrors/better_errors) to work its magic
- a user’s Invitation page was not showing the RSVP buttons
- an organiser should be able to manually RSVP someone without the need for a tutorial, for the cases of an extra student showing up

### CODE REVIEW

File: app/models/member.rb [hunk 1]
```diff
@@ -48,9 +48,13 @@ def banned_permanently?
     bans.permanent.present?
   end
 
+  def name_and_surname
+    [name, surname].compact.join ' '
+  end
```
Comment (gnclmorais) [inline app/models/member.rb:53]: I found it handy to have a method without pronouns for the organisers’ interface, for brevity.

File: app/views/admin/workshops/changes.html.haml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.container
+  %p
+    This page is useful to see all the invitations acted upon, meaning any people
+    that RSVPed attenting or not attending.
```
Comment (matyikriszta) [inline app/views/admin/workshops/changes.html.haml:4]:
```suggestion
    This page is useful to see all the invitations acted upon, meaning any people
    that RSVPed attending or not attending.
```

Small typo in this sentence

File: spec/features/admin/manage_workshop_attendances_spec.rb [hunk 3]
```diff
@@ -67,5 +67,38 @@
...
       expect(page).to have_content(invitation.tutorial)
     end
+
+    context '#changes' do
+      before do
+        # Workshop invitations without `attending` status
+        Fabricate(:workshop_invitation, workshop: workshop, role: 'Coach')
+        Fabricate(:workshop_invitation, workshop: workshop, role: 'Student')
+
+        # Not attenting
```
Comment (matyikriszta) [inline spec/features/admin/manage_workshop_attendances_spec.rb:77]:
```suggestion
        # Not attending
```

**Review by matyikriszta — COMMENTED** [review]
@gnclmorais code looks good overall, noticed two small typos. Going to check the changes out and test locally before I approve.

**Review by matyikriszta — APPROVED** [review]
@gnclmorais I've reviewed this locally and all looks good! Once those two typos are fixed this is good to ship 🚢

**Comment by gnclmorais** [comment]
Thanks for fixing the typos! Sorry it took me a while to get back to this…

**Pull request merged**